rename.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
-charm=$(grep -E "^name:" metadata.yaml | awk '{print $2}')
```

Comment:
It would be nice to eventually remove the rename.sh script altogether actually.  This will be especially important if/when we split build-on/run-on, so we'll have multiple output charms (one for each supported ubuntu release).